src/nix/build.cc

```diff
@@ -41,6 +41,32 @@ nlohmann::json builtPathsWithResultToJSON(const std::vector<BuiltPathWithResult>
     return res;
 }
 
+void createOutLinks(const Path& outLink, const std::vector<BuiltPathWithResult>& buildables, ref<Store>& store) {
+    auto store2 = store.dynamic_pointer_cast<LocalFSStore>();
```

Comment:
I think it is better to do the downcast in the caller.
```suggestion
void createOutLinks(const Path& outLink, const std::vector<BuiltPathWithResult>& buildables, LocalFSStore & store)
{
```
Also we usually do the opening brace on the next line.

Reply:
The opening brace is now on a new line and the downcast is at the call-site.